harness/determined/pytorch/_pytorch_context.py

```diff
@@ -35,7 +34,7 @@ def get_model(self) -> torch.nn.Module:
         check.check_not_none(self.model)
         return cast(torch.nn.Module, self.model)
 
-    def get_optimizer(self) -> torch.optim.Optimizer:  # type: ignore
+    def get_optimizer(self, idx: Optional[int] = None) -> torch.optim.Optimizer:  # type: ignore
```

Comment:
non-blocking: until we make the new API public, I think expanding this arg might be somewhat confusing.